playlist.md

```diff
@@ -39,4 +40,4 @@
 - [004: RBAC](episodes/004/README.md)
 - [003: Istio](episodes/003/README.md)
 - [002: Networking and Services](episodes/002/README.md)
-- [001: A Quick Tour](episodes/001/README.md)
\ No newline at end of file
+- [001: A Quick Tour](episodes/001/README.md)
```

Comment:
for some reason `vim` decided to change this line, but it looks good:
 
https://github.com/p0lyn0mial/tgik/blob/playlist_episode_41/playlist.md